unit_tests/Common.d

```diff
@@ -610,53 +631,105 @@ void assertRunsDStep(
             auto extra = findExtraGNUStepPaths(file, line);
 
             if (extra.empty)
-                return;
+                throw new NoGNUStepException();
```

Comment:
The code was used in a single place before. Now the function is reused in two places. I used exception to communicate the outer functions that the GNUStep is no available, so that thay can return silently (as before).